modules/local/virusbreakend/meta.yml

```diff
@@ -0,0 +1,63 @@
+name: virusbreakend
+description: Detect viral integration from WGS data
+keywords:
+  - viral
+  - integration
+tools:
+  - virusbreakend:
+      description: Performs detection of viral intergation from WGS data
+      homepage: https://github.com/PapenfussLab/gridss/blob/master/VIRUSBreakend_Readme.md
+      documentation: https://github.com/PapenfussLab/gridss/blob/master/VIRUSBreakend_Readme.md
+      licence: ["GPL v3"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [id: 'sample_id']
+  - bam:
+      type: file
+      description: BAM file
+      pattern: "*.{bam}"
+  - gridss_config:
+      type: file
+      description: GRIDSS configuration file (optional)
```

Comment:
Why is the optional file given second?

Reply:
Ordering was driven by the process invoke where the a variable representing the GRIDSS config filepath is always provided, and that specific sort order seemed most natural. Though now you point this out from that perspective it makes sense to adjust.

- I've made this change in https://github.com/nf-core/oncoanalyser/pull/67